### ISSUE TITLE
stuck in a T shape

### ISSUE THREAD

**Issue by monstermutt** [nicoco007/DynamicOpenVR#15]
For some odd reason after downloading and doing everything, my avatar when launching the game is stuck in a T pose, and i cant access the mods menu within the game. The avatar doesn't move and i dont know how to fix that.

**Comment by nicoco007** [comment]
Could you share your latest log? You'll have to put it in a ZIP to upload it on GitHub.

**Comment by Payne05** [comment]
i have a stuck in t pose problem too i created a avatar but it always stand not moveing in t pose https://pastebin.com/RXDT0kq5 the avatar called Edgy thats not moveing

**Comment by simba-limba** [comment]
My controllers do not track with the newest custom avatar keeping me in a t shap other than my head

**Comment by katafox** [comment]
I used custom avatars quite a while ago, and back then the avatars would move. Now, I seem to be experiencing the same issue where the avatar doesn't move with my hands. I have verified that I have the correct version installed and followed the troubleshooting to check that the controllers were using the correct binding. I'm attaching my latest log since that was the last thing asked for when this thread was active. I appreciate any advice on how to proceed fixing it :)
[_latest.log](https://github.com/nicoco007/DynamicOpenVR/files/4316480/_latest.log)

**Comment by nicoco007** [comment]
@katafox Could you attach the full latest log instead of the latest DynamicOpenVR one? Unexpected errors usually pop up in that one.

**Comment by katafox** [comment]
By full latest log do you just mean the one in just the /logs? Hopefully this is the one you need.
[_latest.log](https://github.com/nicoco007/DynamicOpenVR/files/4322061/_latest.log)

**Comment by nicoco007** [comment]
dammit bot

**Comment by nicoco007** [comment]
Looks like you're using a rather old version of DynamicOpenVR (0.1.0 beta 3). Could you try using the [latest version](https://github.com/nicoco007/DynamicOpenVR/releases/tag/v0.1.1) (0.1.1) ?

**Comment by katafox** [comment]
That is strange. I know I started with an older version, but in my own troubleshooting I had found the 0.1.1 release. I overwrote the files and it still didn't work, so I decided to uninstall beatsaber and start fresh. I doublechecked that I unzipped the 0.1.1 release along with the BeatSaberCustomAvatars release, and n
[_latest.log](https://github.com/nicoco007/DynamicOpenVR/files/4325950/_latest.log)
ow the mod isn't showing up at all. I'm reattaching my latest log.
